### DISCUSSION TITLE
wait who's Will?

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#815] [(A) Anything]
I've been using this for a long time but I never understood one thing...

Why is it called "Wills Cheat Menu"?

I still haven't seen anyone named will

**Comment by ghost** [comment]
will is melncat who retired

**Reply by ghost** [reply]
I am new here I just tested the hacks :D

**Reply by NX721** [reply]
Ok, but I **Test hacks** as well (more otfen)

**Reply by ghost** [reply]
> Ok, but I **Test hacks** as well (more otfen)

does that mean anything?

also you wrote "otfen" instead of often

**Reply by ghost** [reply]
> does that mean anything?

well hes a contributor

**Reply by ghost** [reply]
O K

**Reply by PatheticMustan** [reply]
"Pathetic-Burger"...?

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/81431824/112987376-a7868200-9130-11eb-8a13-e3e05f90a527.png)
 
@PatheticMustan This is a Pathetic Burger

**Reply by ghost** [reply]
I found the image on google by the way XD
